canonical-web-and-design/react-components
# Add empty table state to MainTable

## Done

- Add empty table state to MainTable for when data is waiting on or has no data
- Driveby: fix linting error in Accordion.js

## QA

- Pull branch
- Run `yarn run serve`
- View Storybook at http://localhost:8403/
- Click to MainTable component
- View 'Empty' state
- Sanity check code
